chore(docs): weekly review (#4654)
scaleway/docs-content: pages/webhosting/reference-content/classic-hosting-migration-information.mdx

### What will my new hosting plan be?

> After [initiating the migration](/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You will then have the possibility to select a Scaleway Web Hosting plan, tailored to meet your specific requirements.

Reviewer: ```suggestion
After [initiating the migration](/webhosting/how-to/migrate-webhosting-plan/) of your Online.net hosting plan, it will be transferred to Scaleway. During this transition, your existing offer will be temporarily moved to a migration plan. You can then select a Scaleway Web Hosting plan, tailored to meet your specific requirements.
```
